javascript/javascript1/week2/script_2.js

```diff
@@ -0,0 +1,82 @@
+//*EVENT APPLICATION
+
+let todayDay = new Date();
+
+let getEventWeekday = [
```

Comment:
When using the prefix `get`, `fetch`, `handle`, `on` one usually is referring to a function call.

Here I would call the array `weekdays`